Makefile

```diff
@@ -267,7 +267,7 @@ app-interface-push-images:
 			-e QUAY_USER=${QUAY_USER} \
 			-e QUAY_TOKEN=${QUAY_TOKEN} \
 			"${IMAGE_ORG}/${IMAGE_NAME}:${VERSION}" \
-			make push-images; \
+			CONTAINER_RUNTIME=docker make push-images; \
```

Comment:
I think you want:
```sh
-e CONTAINER_RUNTIME=${CONTAINER_RUNTIME} \
make push-images; \
```

Testing locally with something like `podman run --rm busybox VAR=value env` just causes the container runtime to try and execute `VAR=value`.

Reply:
Hmm, actually this `source hack/determine-container-runtime.sh` should probably move to the `push-images` target to ensure the correct runtime is picked up within the `app-interface-push-images` container.